packages/syntax-core/src/Button/Button.module.css

```diff
@@ -0,0 +1,143 @@
+.button {
+  border: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.buttonGap {
+  gap: 4px;
+}
+
+.button:hover {
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0.1) 0%,
+    rgba(0, 0, 0, 0.1) 100%
+  );
+  transition-duration: 0.2s;
+  cursor: pointer;
+}
+
+.button:focus-visible {
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0.2) 0%,
+    rgba(0, 0, 0, 0.2) 100%
+  );
+  box-shadow: 0 0 0 4px #000;
+  outline: solid 2px #fff;
+}
+
+.button:active {
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0.2) 0%,
+    rgba(0, 0, 0, 0.2) 100%
+  );
+  transform: scale(0.97);
+  transition-duration: 0.2s;
+}
+
+.button:disabled {
+  filter: opacity(30%);
+  background-image: none;
+  transform: none;
+  cursor: auto;
+}
+
+.fullWidth {
+  width: 100%;
+}
+
+.sm {
+  min-width: 52px;
+  height: 32px;
+  padding-left: 12px;
+  padding-right: 12px;
+  border-radius: 12px;
+}
+
+.md {
+  min-width: 72px;
+  height: 40px;
+  padding-left: 16px;
+  padding-right: 16px;
+  border-radius: 12px;
+}
+
+.lg {
+  min-width: 88px;
+  height: 48px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-radius: 16px;
+}
+
+.smIcon {
```

Comment:
if we're inlining the svgs, will we need this stuff still because we should be able to tweak these directly?

Reply:
@somethiiing For now we are still using Material UI icons. I noticed that they have more than 2000 icons in 5 different variations so it would be a lot of work to maintain those